### ISSUE TITLE
Paged results

### ISSUE THREAD

**Issue by garrjo** [MightyOrm/Mighty#21]
I'm attempting to write a Paged Query and I have configured it to do so; however, the yielded result set is atypical of what normally would occur in a result set coming from MighyOrm.  Do we have more documentation to give me an example of how exactly is the best practice to work wiith a PagedResult?
`
            var tbl = new Mighty.StartingAccounts();
            var page = tbl.Paged(currentPage: currentPage, pageSize: 100, columns:"Id");
`
As I'd like to work with them in a list I'm unfamiliar with how to bring them to it?  Since the result is PagedResults<dynamic>  ... Can you give me a hint here?

**Comment by mikebeaton** [comment]
I was hoping this should be fairly self explanatory based on IntelliSense?

The `PagedResults` object has fields `Items`, `TotalPages` and `TotalRecords`. If you just type `.` next to a `PagedResults` object in Visual Studio or VS Code then it should show them to you as auto-complete options? And the fields themselves have IntelliSense.

Does that help? Is IntelliSense all working for you? It should be since [v. 3.0.5](https://github.com/MightyOrm/Mighty/tree/3.0.5) and 66f0106 .

**Comment by garrjo** [comment]
It should but, traction seems to vary.   I will take a look at next run

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by mikebeaton** [comment]
@garrjo I'm gathering that what would have helped here is just a bit more detail in the current [docs on paging](https://mightyorm.github.io/Mighty/docs/paging.html), right?

**Comment by garrjo** [comment]
Absolutely,. Having an example to go by beyond the implementation.   A Use case would definitely given direction as to how exactly to implement

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by mikebeaton** [comment]
Fair point! :)

I'm trying to match - but not necessarily exceed, given available time and resources! - the detail of the docs for Massive. So I agree I should document the fields that are in `PagedResults` (in the docs, not just in IntelliSense!), and also the reason for having them; namely, that you can't show a user paging info unless the call to the paging method returns the total number of pages/items along with the items for the current page.

So I'll leave this Issue open pending adding this info to the docs.

**Comment by garrjo** [comment]
Sounds great.  Thank you for your work and time.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________
